src/Morphir/IR/NodeId.elm

```diff
@@ -1,34 +1,60 @@
 module Morphir.IR.NodeId exposing (..)
 
 import Morphir.IR.FQName as FQName exposing (FQName)
+import Morphir.IR.Module exposing (ModuleName)
+import Morphir.IR.Name as Name
+import Morphir.IR.Path as Path exposing (Path)
 
 
 type NodeID
     = TypeID FQName
     | ValueID FQName
+    | ModuleID ModuleName
```

Comment:
The package name should also be included in case we want to decorate an upstream package instead ofg the current one.